src/modules/SX127x/SX127x.h

```diff
@@ -876,6 +876,13 @@ class SX127x: public PhysicalLayer {
     */
     int16_t setPreambleLength(size_t preambleLength) override;
 
+    /*!
+      \brief Sets FSK preamble polarity. Allowed values are RADIOLIB_SX127X_PREAMBLE_POLARITY_55 or RADIOLIB_SX127X_PREAMBLE_POLARITY_AA. Not allowed in LoRa mode.
+      \param polarity Preamble polarity bit in FSK mode).
+      \returns \ref status_codes
+    */
+    int16_t setPreamblePolarity(int polarity);
```

Comment:
Please add `setPreamblePolarity` to keywords.txt

Reply:
done